src/main/java/com/tabnine/inline/InlineCompletionHandler.java

```diff
@@ -69,6 +75,16 @@ public void retrieveAndShowCompletion(
       return;
     }
 
+    if (lastShownSuggestion != null) {
+      // this means that the user did not type as suggested -
+      // we couldn't find completions in the cache, but there was a suggestion rendered
+      SuggestionDroppedReason reason =
+          completionAdjustment instanceof LookAheadCompletionAdjustment
+              ? SuggestionDroppedReason.ScrollLookAhead
+              : SuggestionDroppedReason.UserNotTypedAsSuggested;
```

Comment:
@yairco1990 this is how i differentiate `ScrollLookAhead` from `UserNotTypedAsSuggested` - wdyt?

Reply:
lgtm